storage-node/src/services/webApi/app.ts

```diff
@@ -125,109 +121,11 @@ export async function createApp(config: AppConfig): Promise<Express> {
         message: err.message,
       })
     }
-
-    next()
   })
 
   return app
 }
 
-/**
```

Comment:
Since we are getting rid of these.. we can also drop

```ts
    // when enabling upload auth ensure the keyring has the operator role key and set it here.
    const enableUploadingAuth = false
    const operatorRoleKey = undefined
```

in `server.ts` and the corresponding fields in the `AppConfig` type

Does it make sense then to also remove the relevant parts in the api spec `openapi.yaml`?

Reply:
Done in [fd44d0d](https://github.com/Joystream/joystream/pull/5001/commits/fd44d0d3c38c0b28b4785694aed766920aca9797). Also removed relevant parts from the API spec file.